mentat/cost_tracker.py

```diff
@@ -44,6 +44,9 @@ def log_api_call_stats(
         costs_logger = logging.getLogger("costs")
         costs_logger.info(speed_and_cost_string)
 
+    def log_whisper_call_stats(self, seconds: float):
```

Comment:
Hmm, this makes me think we should make a separate function for logging vs displaying; definitely not relevant to this PR though!

Reply:
I separated it out because the other one had token counting as a central part of it's logic. If it was a little more expensive I'd want to display after transcription but at less than a penny for a whole minute it doesn't seem worth.